constructor_io/modules/autocomplete.py

```diff
@@ -5,20 +5,21 @@
 
 import requests as r
 
+from constructor_io.helpers.exception import ConstructorException
 from constructor_io.helpers.utils import (clean_params, create_auth_header,
                                           create_request_headers,
                                           create_shared_query_params,
                                           throw_http_exception_from_response)
 
 
-def create_autocomplete_url(query, parameters, user_parameters, options):
+def _create_autocomplete_url(query, parameters, user_parameters, options):
```

Comment:
Mark methods as "weak internal use" so they are private and do not appear in docs

Reply:
Love these comments - thank you!